lib/constants/collectives.js

```diff
@@ -10,6 +10,13 @@ export const CollectiveType = {
   FUND: 'FUND',
 };
 
+export const AccountTypesWithHost = [
+  CollectiveType.COLLECTIVE,
+  CollectiveType.EVENT,
+  CollectiveType.FUND,
+  CollectiveType.PROJECT,
+];
```

Comment:
Isn't this just a better name for `CollectiveFamilyTypes`?

Reply:
It is indeed, I'll rename that in the other palces